internal/utils/helper.go

```diff
@@ -39,15 +35,21 @@ func BoolToPtr(b bool) *bool {
 	return &b
 }
 
+var localImportRegex = regexp.MustCompile(`(?ims)^\s*IMPORT[\s(]+.+FROM\s+LOCAL\s+CSV.*$`)
+
 func IsImportQuery(query string) bool {
 	return localImportRegex.MatchString(query)
 }
 
+const ROW_SEPARATOR_PLACEHOLDER = "RowSeparatorPlaceholder"
+
+var rowSeparatorQueryRegex = regexp.MustCompile(fmt.Sprintf(`(?i)(ROW\s+SEPARATOR\s+=\s+(["|'])?(?P<%s>[a-zA-Z]+)(["|']?))`, ROW_SEPARATOR_PLACEHOLDER))
```

Comment:
Would it be possible to split the regex into multiple lines and add some comments?
Esp. maybe also about the algo / approach?
And maybe on the modifiers, `(?i)` (case-insensitive, I guess)